pkg/controller/master/virtualmachine/vmi_controller.go

```diff
@@ -17,11 +17,29 @@ import (
 
 const (
 	VirtualMachineCreatorNodeDriver = "docker-machine-driver-harvester"
+
+	AnnotationTopologyRegion = "topology." + builder.HarvesterAPIGroup + "/region"
+	AnnotationtopologyZone   = "topology." + builder.HarvesterAPIGroup + "/zone"
+	AnnotationTopologyHost   = "topology." + builder.HarvesterAPIGroup + "/host"
 )
 
 // hostLabelsReconcileMapping defines the mapping for reconciliation of node labels to virtual machine instance annotations
-var hostLabelsReconcileMapping = []string{
-	v1.LabelTopologyZone, v1.LabelTopologyRegion, v1.LabelHostname,
+var hostLabelsReconcileMapping = []struct {
+	nodeLabel     string
+	vmiAnnotation string
+}{
+	{
+		nodeLabel:     v1.LabelTopologyZone,
+		vmiAnnotation: AnnotationTopologyRegion,
```

Comment:
Should be Zone->Zone

Reply:
should we also use the customized keys instead of the v1 labelTopology to make it consistent?